tracker/internal/services/service_search.go

```diff
@@ -239,7 +239,71 @@ func (s *searchService) BreadthFirstSearch(server tracker.SearchService_BreadthF
 }
 
 func (s *searchService) DepthFirstSearch(server tracker.SearchService_DepthFirstSearchServer) error {
```

Comment:
Haven't been able to test this yet. Need some help on setting up deps cli locally, so I could test this.

Reply:
deps does produce a docker image now so you should be able to invoke it through the container. You can probably get clever and set the container up on the same network as the compose stack you're using. Usually `{dirname}_default` unless otherwise specified.

```
$ docker network ls
NETWORK ID          NAME                  DRIVER              SCOPE
fbcf114e6803        bridge                bridge              local
681faede97a8        cockroachdb_default   bridge              local <---
96db5116e130        host                  host                local
97d777e6bad8        mariadb_default       bridge              local <---
17b755f69775        none                  null                local
724585ee008e        sqlite_default        bridge              local <---
```